daemon/appinfo.c

```diff
@@ -59,6 +59,22 @@ typedef enum
     APPINFO_STATE_DELETED,
 } appinfo_state_t;
 
+typedef enum
+{
+    APPINFO_FILE_UNCHANGED,  // Nothing has changed
+    APPINFO_FILE_CHANGED,    // File timestamp has changed
+    APPINFO_FILE_INVALID,    // File is inaccessible => error
+    APPINFO_FILE_DELETED,    // File existed but was removed
+    APPINFO_FILE_MISSING,    // File never existed
```

Comment:
"never say never"... Unless I'm mistaken it would be "was not present in last scan?"